village/src/App.js

```diff
@@ -14,12 +15,31 @@ class App extends Component {
   // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
   // Notice what your map function is looping over and returning inside of Smurfs.
   // You'll need to make sure you have the right properties on state and pass them down to props.
+  componentDidMount = () => {
+    axios.get('http://localhost:3333/smurfs')
+    .then (response => {
+      this.setState({smurfs: response.data});
+    })
+    .catch(err => {
+      console.log(err);
+    })
+  };
   render() {
     return (
       <div className="App">
-        <SmurfForm />
-        <Smurfs smurfs={this.state.smurfs} />
+        <ul className='NavBar'>
+          <li><Link to='/'>HOME</Link></li>
+          <li><Link to='/smurf-form'>Add a Smurf</Link></li>
+         </ul>
+          <Route exact path='/smurf-form' component={SmurfForm} />
```

Comment:
You'll want to use `render = { props => <Component />}` so that you can pass properties to the component nested in your route.